pkg/virt-controller/watch/clone/clone.go

```diff
@@ -246,10 +247,10 @@ func (ctrl *VMCloneController) syncSourceVMTargetVM(key string, source *k6tv1.Vi
 
```

Comment:
I'm not sure about that. Some environments don't change the verbosity level and then it's very hard to debug without any log prints.

When the tests aren't flakie anymore this shouldn't spam the logs. Let's hope that's the case after this PR :)

Reply:
I dont think its related to this PR really, since its in a different phase of the clone, this print will still be printed until the restore is done, and since there is a print after it if you dont see it you can pretty much understand where you stand. but ok we can decide about it in the future